data/index.js

```diff
@@ -80,18 +80,25 @@ function addAssignment(assignmentName, prompt, dueDate) {
 	if (typeof dueDate != "string") {
 		return Promise.reject("Due date must be provided");
 	}
+	return courses().then(collection => {
+		return collection.findOne({_id: classID}).then((course) => {
+			return course.studentIDs;
+		});
+	}).then((ids) => {
+		let submissions = ids.map((x) => {return {studentID: x, grade: 'NA'}});
```

Comment:
This functionality is actually already taken care of in the "addAssignmentToCourse" function. Do you have a particular reason for wanting to move it here?